app/workers/hard_worker.rb

```diff
@@ -0,0 +1,14 @@
+class HardWorker
+  include Sidekiq::Worker
+
+  def perform(user)
```

Comment:
Be carefull with the methods and params names. Someone can confuse your param user with a user instance and in this case its an email.